src/main/scala/com/mozilla/telemetry/streaming/CrashesToOpenTsdb.scala

```diff
@@ -0,0 +1,32 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+package com.mozilla.telemetry.streaming
+
+import com.mozilla.telemetry.sinks.BatchHttpSink
+
+object CrashesToOpenTsdb extends CrashPingStreamingBase {
+
+  override val sparkAppName: String = this.getClass.getSimpleName
+
+  override def buildOutputString(measurementName: String, timestamp: Long,
+                                 buildId: String, tags: Map[String, String]): String = {
+    val formattedTags = tags
+      // Can't escape quotes in string interpolation: https://github.com/scala/bug/issues/6476
+      .map { case (k, v) => List(k, ": ", v).mkString("\"", "\"", "\"") }
```

Comment:
I suggest to either use triple quotes here:
```
s""""$k": "$v""""
```
or use json4s to build json from map, like in https://github.com/mozilla/telemetry-streaming/blob/master/src/main/scala/com/mozilla/telemetry/streaming/ExperimentEnrollmentsToTestTube.scala#L114-L121